internal/client/graph_api.go

```diff
@@ -49,16 +61,29 @@ func NewGraphAPI(options GraphAPIOptions) *GraphAPI {
 	}
 }
 
+// In order for all importers ran at the same time to not read again at the same time we want to add a random duration to the
+// anti entropy duration of 1/4 of the duration.
+func (gapi *GraphAPI) isAntiEntropyDurationExceeded() bool {
+	delta := int64(gapi.options.AntiEntropyDuration) / 4
+	max := delta
+	min := -delta
+	randDelta := time.Duration(rand.Int63n(max-min+1) + min)
```

Comment:
I don't understand why you need min and max there, `max-min` is just max*2. 
This could be rewritten as :
```
	quarter := int64(gapi.options.AntiEntropyDuration) / 4
	randDelta := time.Duration(rand.Int63n(quarter * 2) - quarter)
```

Reply:
Sure